mcs/tools/mkbundle/mkbundle.cs

```diff
@@ -1085,8 +1089,10 @@ static void GenerateBundles (List<string> files)
 			tc.WriteLine ("\n}\n");
 			tc.WriteLine ("#endif\n");
 
-
-			tc.WriteLine ("static char *image_name = \"{0}\";", prog);
+			if (no_managed_entry_point)
+				tc.WriteLine ("static char *image_name = NULL;", prog);
```

Comment:
Could/should just be `tc.WriteLine("static char* image_name");` The extra `prog` even runs successfully?

Reply:
Oh good catch, yeah that was a copy-paste error.